src/orgasm_control.c

```diff
@@ -298,6 +300,8 @@ static void orgasm_control_updateEdgingTime() { // Edging+Orgasm timer
     if (orgasm_control_isPermitOrgasmReached() && !orgasm_control_isPostOrgasmReached()) {
         if (output_state.control_motor) {
             orgasm_control_pauseControl(); // make sure orgasm is now possible
+            // Calculate motor increment once, from here to end of post orgasm
+            output_state.motor_increment = calculate_increment(Config.motor_start_speed, Config.motor_max_speed, Config.motor_ramp_time_s);
```

Comment:
Don't cache this value in a global state. Apologies, when I said a "higher scope" I meant at the top of the function call you were in.

Config values can change at runtime, and this value should be recalculated whenever it is used to ensure the freshest config values. There is no need to devote a static memory allocation to this value. I just did not want the value recalculated between your conditional clause and the final usage of it.